.codecov.yml

```diff
@@ -0,0 +1,6 @@
+coverage:
+  branch: main
```

Comment:
I saw in this link: https://docs.codecov.io/docs/commit-status#branches
that the yaml looks like:
```yml
coverage:
  status:
    project:
      default:
        threshold: 1%
        branches: 
          - main
```
But the current change also works, I don't know if there is difference. I guess it's fine.

Reply:
I added that per https://docs.codecov.io/docs/codecov-yaml#master-copy

Since we have a `main`, not a `master`, branch, I want to make sure that codecov picks the correct branch to cache its YAML